COMPILING

```diff
@@ -1,267 +0,0 @@
-
-0) Run `cmake --version`. If you get a command-not-found error, or the installed
-   version is lower than 3.2, go and install a new version. Most Linux
-   distributions have a package for CMake, and Mac users can get it through
-   Homebrew.  Windows users should download it manually from cmake.org.
-
-1) Create a directory to store your build:
-   `mkdir build`
-   Run this from the *top level* folder of the project. This is different from
-   the other builds, which require you to `cd src` first.
-
-2) Generate build files with CMake:
-   `cmake -H. -Bbuild`
-   This command tells CMake to use the configuration in the current directory,
-   and to generate build files into the `build` directory.
-   This is the point to specify custom build settings, such as compilers and
-   build back-ends. You can use clang (for example) by adding the argument
-   `-DCMAKE_CXX_COMPILER=clang++` to the command line. You can also tell
-   CMake to generate IDE projects by supplying the `-G` flag.
-   Run `cmake -G` for a comprehensive list of supported back-ends.
-
-   Generally it is not necessary to manually specify individual compiler or
-   linker flags, as CMake defines a number of "build modes" including Debug
-   and Release modes. To build in a particular mode, add the flag
-   `-DCMAKE_BUILD_TYPE=Debug` (or `Release`) to the initial invocation.
-
-   If you *do* need to manually add flags, use `-DCMAKE_CXX_FLAGS=...` and
-   `-DCMAKE_EXE_LINKER_FLAGS=...`. This is useful for enabling clang's
-   sanitizers.
-
-   Finally, to enable building universal binaries on macOS, you can pass the
-   flag `-DCMAKE_OSX_ARCHITECTURES=i386;x86_64`. If you don't supply this flag,
-   the build will just be for the architecture of your machine.
-
-3) Run the build:
-   `cmake --build build`
-   This command tells CMake to invoke the correct tool to run the build in the
-   `build` directory. You can also use the build back-end directly by invoking
-   `make`, `ninja`, or opening the generated IDE project as appropriate.
-
-
-WORKING WITH ECLIPSE
---------------------
-
-To work with Eclipse, do the following:
-
-1) Select File -> New -> Makefile Project with Existing Code
-
-2) Type "cprover" as "Project Name"
-
-3) Select the "src" subdirectory as "Existing Code Location"
-
-4) Select a toolchain appropriate for your platform
-
-5) Click "Finish"
-
-6) Select Project -> Build All
-
-
-CODE COVERAGE
--------------
-
-Code coverage metrics are provided using gcov and lcov. Ensure that you
-have installed lcov from http://ltp.sourceforge.net/coverage/lcov.php
-note for ubuntu lcov is available in the standard apt-get repos.
-
-To get coverage metrics run the following script from the regression
-directory:
-
-  get_coverage.sh
-
-This will:
- 1) Rebuild CBMC with gcov enabled
- 2) Run all the regression tests
- 3) Collate the coverage metrics
- 4) Provide an HTML report of the current coverage
```

Comment:
Bundling a file rename along with a change to the content seems somewhat poor form as it makes it hard to see what is changed.

Reply:
If I look at 1dcc82c github shows line-by-line changes, so I'm not sure this is a big issue.

Reply:
Weird that github does this when looking at a single commit, but doesn't get it when looking at all the commits in a PR.

Reply:
OK; may just be a UI issue then.